pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -205972,8 +205972,6 @@ self: {
        ];
        description = "Treemap visualiser for GHC prof files";
        license = stdenv.lib.licenses.bsd3;
-       hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
You don't need to remove these, they will automatically be removed by `hackage2nix` before the next time the `haskell-updates` branch is merged.